h2o-extensions/target-encoder/src/main/java/ai/h2o/targetencoding/TargetEncoderBuilder.java

```diff
@@ -55,9 +60,8 @@ public void computeImpl() {
       disableIgnoreConstColsFeature();
       TargetEncoderModel.TargetEncoderOutput output = new TargetEncoderModel.TargetEncoderOutput(TargetEncoderBuilder.this, _targetEncodingMap, priorMean);
       _targetEncoderModel = new TargetEncoderModel(_job._result, _parms, output, tec);
-
-      _targetEncoderModel.write_lock(_job);
-      _targetEncoderModel.unlock(_job);
```

Comment:
@michalkurka  What is the purpose of these two lines? Does it make sure that `_targetEncoderModel` is updated globally?